tubearchivist/home/src/es/index_mapping.json

```diff
@@ -62,7 +84,7 @@
                             "type": "boolean"
                         },
                         "integrate_sponsorblock": {
-                            "type" : "boolean"
+                            "type": "boolean"
```

Comment:
please keep the json formatting as is. I know, people disagree but for me it looks strange to have a space before the `:`.